pkgs/servers/shairport-sync/default.nix

```
@@ -21,15 +49,26 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ autoreconfHook pkg-config ];
 
   buildInputs = [
-    openssl
+    openssl_1_1
     avahi
-    alsa-lib
-    libdaemon
     popt
     libconfig
-    libpulseaudio
-    soxr
-  ] ++ optional stdenv.isLinux glib;
+  ]
+  ++ (optional enableLibdaemon libdaemon)
+  ++ (optional enableAlsa alsa-lib)
+  ++ (optional enablePulse libpulseaudio)
+  ++ (optional enablePipewire pipewire)
+  ++ (optional enableJack libjack2)
+  ++ (optional enableSoxr soxr)
+  ++ (optionals enableAirplay2 [
+    libplist
+    libsodium
+    libgcrypt
+    libuuid
+    ffmpeg
+    vim # for xxd
```

unixtools.xxd is an alias for vim but I think we want to use that here instead incase that ever changes

```suggestion
    unixtools.xxd
```